index.js

```diff
@@ -0,0 +1,44 @@
+const form = document.querySelector('.add-book-form');
+const bookTitle = document.querySelector('#book-title');
+const bookAuthor = document.querySelector('#book-author');
+const booksContainer = document.querySelector('.books-container');
+
+let Books = [];
+
+if (localStorage.getItem('Books')) {
+  Books = JSON.parse(localStorage.getItem('Books'));
+}
+
+const addBook = (e) => {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you use a unique id property for each book since it would make it easier to remove books with duplicate values. 

(see code example):
`const books = [{title: 'title', author: 'author', id: 'unique id'}]; // example of the book collection data structure`

Reply:
I will make these changes. Thank You @topeogunleye